guides/theming.md

```diff
@@ -235,6 +238,95 @@ your project's `angular.json` file][adding-styles].
 
 [adding-styles]: https://angular.io/guide/workspace-config#styles-and-scripts-configuration
 
+#### Theming dimensions
+
+Angular Material themes are divided along several dimensions:
+
+Base
+: Common base styles for the design system. These styles don't change based on your configured
+colors, typography, or density, so they only need to be included once per application. These
+styles may include things such as border-radius, border-width, etc. All components have a base
+mixin that can be used to include its base styles. (For example,
+`@include mat.checkbox-base($theme)`)
+
+Color
+: Styles related to the colors in your application. These style should be included at least once in
+your application. Depending on your needs, you may need to include these styles multiple times
+with different configurations. (For example, if your app supports light and dark theme colors.)
+All components have a color mixin that can be used to include its color styles. (For example,
+`@include mat.checkbox-color($theme)`)
+
+Typography
+: Styles related to the fonts used in your application, including the font family, size, weight,
+line-height, and letter-spacing. These style should be included at least once in your application.
+Depending on your needs, you may need to include these styles multiple times with different
+configurations. (For example, if your app supports reading content in either a serif or sans-serif
+font.) All components  have a typography mixin that can be used to include its typography
+styles. (For example, `@include mat.checkbox-typography($theme)`)
+
+Density
+: Styles related to the size and spacing of elements in your application. These style should be
+included at least once in your application. Depending on your needs, you may need to include these
+styles multiple times with different configurations. (For example, if your app supports both a
+normal and compact mode). All components have a density mixin that can be used to include its
+density styles. (For example, `@include mat.checkbox-density($theme)`)
+
+All components also support a theme mixin that can be used to include the component's styles for all
+theme dimensions at once. (For example, `@include mat.checkbox-theme($theme)`).
+
+If you need to include [multiple themes in a single file](defining-multiple-themes), the recommended
+approach to structuring the file is to include the full theme mixin for any components used in your
+app at the root, and then have scoped overrides of individual theme dimensions as needed. For
+example:
+
+```scss
+@use '@angular/material' as mat;
```

Comment:
Should I include this example or just link to the `defining-multiple-themes` section? It feels a bit redundant, but I'm just trying to get the point across that most people shouldn't have a need to call the `base` mixins directly, since they should be calling the `theme` mixins

Reply:
I'd probably go with just linking